tags/network_architecture.yaml

```diff
@@ -219,6 +219,9 @@ values:
     display_name: Big Transfer
   - id: swin-transformer
     display_name: Swin Transformer
+  - id: vit
```

Comment:
Could you please remove that line again since this item is already defined on line 225?

Reply:
Thank you for pointing it out. I have removed it and committed the change.